tests/components/roku/test_media_player.py

```diff
@@ -56,76 +58,64 @@
 TV_SERIAL = "YN00H5555555"
 
 
-async def test_setup(hass: HomeAssistantType, requests_mock: Mocker) -> None:
+async def test_setup(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
     """Test setup with basic config."""
-    await setup_integration(hass, requests_mock)
+    await setup_integration(hass, aioclient_mock)
 
     entity_registry = await hass.helpers.entity_registry.async_get_registry()
-
     main = entity_registry.async_get(MAIN_ENTITY_ID)
+
     assert hass.states.get(MAIN_ENTITY_ID)
+    assert main
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)